src/lib/transaction_snark_work/transaction_snark_work.mli

```diff
@@ -76,12 +76,7 @@ with type V2.t = t
 type unchecked = t
 
 module Checked : sig
-  type nonrec t = t =
-    { fee : Fee.t
-    ; proofs : Ledger_proof.t One_or_two.t
-    ; prover : Public_key.Compressed.t
-    }
-  [@@deriving sexp, compare, to_yojson]
+  include S
```

Comment:
The `module Stable` should also be removed here. Doesn't need to be done in this PR.

Reply:
👍